lib/Dialect/Torch/Transforms/RefineTypes.cpp

```diff
@@ -293,7 +293,7 @@ class TypeAnalyzer : public ForwardDataFlowAnalysis<ValueKnowledge> {
           ValueKnowledge::getNotNonePessimisticValueState(op->getContext());
       if (input.hasSizes) {
         knowledge.hasSizes = true;
-        knowledge.sizes.resize(input.sizes.size(), kUnknownSize);
```

Comment:
Does using `input.sizes.size()` not work?

Reply:
No